sysdeps/linux/generic/entry.cpp

```diff
@@ -12,6 +13,8 @@ extern "C" void __dlapi_enter(uintptr_t *);
 extern char **environ;
 static mlibc::exec_stack_data __mlibc_stack_data;
 
+size_t __hwcap;
```

Comment:
What is this for?

Reply:
This variable is used in some places to determine if some hardware features can be used. IIRC it is read from auxv. It also exists in the other libc I checked.

Reply:
Is it used inside mlibc or exported and used by user programs?

Reply:
This gets used by mlibc internally, for instance in some fenv functions.